src/main/java/gregtech/api/metatileentity/SimpleMachineMetaTileEntity.java

```diff
@@ -443,7 +449,7 @@ protected ModularUI.Builder createGuiTemplate(EntityPlayer player) {
 
         if (exportItems.getSlots() + exportFluids.getTanks() <= 9) {
             ImageWidget logo = new ImageWidget(152, 63 + yOffset, 17, 17, GTValues.XMAS.get() ? GuiTextures.GREGTECH_LOGO_XMAS : GuiTextures.GREGTECH_LOGO).setIgnoreColor(true);
-            SlotWidget circuitSlot = new SlotWidget(circuitInventory, 0, 124, 62 + yOffset, true, true, false)
+            SlotWidget circuitSlot = new GhostCircuitSlotWidget(circuitInventory, 0, 124, 62 + yOffset)
```

Comment:
Slot should accept only one item
![image](https://user-images.githubusercontent.com/3093101/218300677-9b1d3c5e-6248-410e-84d0-806ef261d692.png)

Reply:
huh, didnt actually know the IC's stack so I didnt try it.

Reply:
Fixed